frontend/src/features/projects/spl/forms/CloseOutFinancialSummaryForm.tsx

```diff
@@ -56,25 +56,29 @@ const CloseOutFinancialSummaryForm = (props: CloseOutFinancialSummaryFormProps)
   const interestComponent = getIn(values, 'interestComponent');
   const salesCost = getIn(values, 'salesCost');
   const netBook = getIn(values, 'netBook');
+  const gainBeforeSpl = getIn(values, 'gainBeforeSpl');
   useEffect(() => {
-    // Calculate the Gain before SPL.
-    setFieldValue(
-      'gainBeforeSpl',
-      calcGainBeforeSpl(
-        getNumber(market),
-        getNumber(interestComponent),
-        getNumber(salesCost),
-        getNumber(netBook),
-      ),
+    const calculatedValue = calcGainBeforeSpl(
```

Comment:
due to duplicated code I had to make the same code change as the projectfinancialsTable here as well.